spec/sanity/deployments.js

```diff
@@ -14,42 +14,80 @@ describe('deployments page', function () {
     });
 
 
+    describe('deleting deployment', function(){
+
+        var deploymentName;
+        var deploymentsOpts;
+
+        beforeEach(function(){
+            deploymentName ='deployment_' + new Date().getTime();
+            deploymentsOpts = {id : deploymentName };
+            // lets deploy a blueprint to make this test rerunable.. I hate it, but what can we do
+            // if deploying was a simple act, we would not care..
+            components.ui.layout.goToBlueprints();
+            components.ui.blueprints.IndexPage.createDeployment({ name : 'nodecellar1' });
+            components.ui.blueprints.CreateDeployment.setDetails({ name : deploymentName , 'raw' : {
+                'agent_private_key_path': 'ggg',
+                'agent_user': 'ggg',
+                'host_ip': 'ggg'
+            }});
+            components.ui.blueprints.CreateDeployment.confirm();
+
+            components.ui.deployments.DeploymentPage.waitForInitializingToStop();
+            browser.sleep(1000).then();
```

Comment:
empty then. 

I know this might not be needed, but for sake of clarity, lets add `done` in function signature, and give it to `then` (see tests that repeat the pattern).